maat-court-data-api/src/main/java/gov/uk/courtdata/contribution/controller/ContributionsController.java

```diff
@@ -113,4 +117,32 @@ public ResponseEntity<ContributionsDTO> create(@Valid @RequestBody CreateContrib
         createContributionsValidator.validate(createContributions);
         return ResponseEntity.ok(contributionsService.create(createContributions));
     }
+
+    @RequestMapping(value = "/{repId}",
+            method = {RequestMethod.HEAD},
```

Comment:
Not sure if HEAD is the correct method to be used here. My understanding is that HEAD should be used to retrieve metadata for a resource that can be retrieved via a GET, where as the implementation here seems to be it's own endpoint solely to retrieve a count I would expect it to be implemented as a GET instead. Happy to discuss though I know there is a lot of use of HEAD elsewhere in the application =)

Reply:
This need conversion with team.